frontend/src/containers/03_products/Products.js

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import axios from "axios";
+
+import "./products.css";
+
+class Products extends React.Component {
+  state = {
+    customerId: 1,
```

Comment:
graag boven de state een comment neerzetten dat customerId voor testing is en die later niet meer gebruikt gaat worden.